data/wanderers middle.txt

```diff
@@ -1766,7 +1766,73 @@ mission "Wanderers: Sestor Search: Varu K'prai"
 			`	The Wanderers don't know what happened next, but the drones are no longer here, so they must have moved on to a different system.`
 				decline
 
-
+mission "Wanderers: Sestor Search: Human Spaceport"
+	landing
+	source
+		government Republic "Free Worlds" Syndicate Neutral
+		attributes north
+	to offer
+		has "Wanderers: Sestor Search: active"
+	on offer
+		conversation
+			`As you bring your ship in for landing at an empty pad, you notice the captain of an adjacent Behemoth inspecting his ship, which is raked with projectile punctures.`
+			choice
+				`	(Approach the captain and try to find out what happened.)`
+					goto approach
+				`	(Ignore him.)
+					decline
+			
+			label approach
+			`	You get a better view of the Behemoth -- the "Heart of Gold", as indicated on the ship's stern -- as you walk up to it. The rear of the ship is badly hammered, and black soot-like stains streak forward across the hull. You can't help but notice that at several projectile entry points along the hull, the metal is bent smoothly inwards, almost as if the penetrator had seared or melted its way through the ship's armor. The captain of the Behemoth looks up from his inspection. Seeing you, he remarks, "We were lucky to get out of that one alive!"`
```

Comment:
Also, just a side note—this is meant to hint at the Korath Repeater Turrets which have heat damage.